src/features/groups/Edit/index.tsx

```diff
@@ -58,9 +53,7 @@ const GroupEdit = ({ groupListItem, onSaved, isAddGroupMembersMode, onGroupMembe
     avatarId: groupListItem?.avatarId
   });
 
-  const [logoSrc, setLogoSrc] = useState<string | null>(groupListItem?.avatarId
-    ? getDownloadFileUrl(groupListItem?.avatarId)
-    : AddAvatarSVG);
+  const [logoSrc, setLogoSrc] = useState<string | null>(getDownloadFileUrl(groupListItem?.avatarId));
```

Comment:
Now we have null instead of AddAvatarSVG as logoSrc if there is no avatarId, right?
I think some image placeholder is better then nothing

Reply:
Well, we shouldn't do that in "business'" components code, things like those should be hidden in appropriate infrastructure places so you can change them once for all usages. in this case, I hid it in Image component implementation, it will pick placeholder in given link is empty, that's my desired behavior. Same code (try create link or replace it with placeholder) were in 3 places, now all job is done in 1 place, if you don't like this approach, please advice something that will still keep code DRY and will distinguish code by it's purpose

Reply:
Oh.. sorry
I missed that you moved placeholder to Image component
Yeah, it is cool
The only thing is maybe to add an ability to pass custom Placeholder component if we need?

Reply:
Good point, will add such opportunity